pkg/core/storage/memory_store.go

```diff
@@ -122,7 +138,8 @@ func (s *MemoryStore) SeekAll(key []byte, f func(k, v []byte)) {
 	s.mut.RLock()
 	defer s.mut.RUnlock()
 	sk := string(key)
-	for k, v := range s.mem {
+	m := s.chooseMap(key)
```

Comment:
Technically, an empty key may be passed to SeekAll, then seek over `mem` map only will be performed. Practically, we don't have such cases.

Reply:
Yeah, I know about this case too, but just wanted to test it out quickly and again, non-prefixed seeks don't make sense to me.